docs/semantic-ui.md

```diff
@@ -0,0 +1,454 @@
+# Semantic UI Integration
+Bear integrates with [Semantic](http://semantic-ui.com/), a UI framework designed for theming. Semantic allows developers to build beautiful websites fast, with concise HTML, intuitive javascript, and simplified debugging. Semantic is responsively designed allowing your website to scale on multiple devices.
```

Comment:
Does Bear really integrate with semantic ui or just bear skin? (question for alban)